graalpy/graalpy-starter/src/main/java/com/example/App.java

```diff
@@ -7,12 +7,11 @@
 package com.example;
 
 import org.graalvm.polyglot.Context;
-import org.graalvm.python.embedding.utils.GraalPyResources;
 
 public class App {
 
     public static void main(String[] args) {
-        try (Context context = GraalPyResources.createContext()) {
+        try (Context context = Context.create("python")) {
```

Comment:
Should we add options for allowing IO/native/threads/etc and importing site? Maybe commented out with explanation "if you need X, uncomment Y"